cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceService.java

```diff
@@ -99,8 +118,6 @@ public DatasetInstanceService(DatasetTypeManager implManager, DatasetInstanceMan
    * @throws IOException if there is a problem in making an HTTP request to check if the namespace exists.
    */
   public DatasetMeta get(Id.DatasetInstance instance, List<? extends Id> owners) throws Exception {
-    // Throws NamespaceNotFoundException if the namespace does not exist
-    ensureNamespaceExists(instance.getNamespace());
```

Comment:
If we are planning on fixing this in a later JIRA, then just comment it out and put the JIRA number here.